src/contractTest/java/uk/gov/hmcts/reform/lrdapi/provider/LrdApiProviderTest.java

```diff
@@ -204,6 +205,7 @@ public void toReturnBuildingLocationDetails() {
     @State({"Region Details exist"})
     public void toReturnRegionDetails() {
         Region region = new Region("2", "London", "");
-        when(regionRepository.findByDescriptionIgnoreCase(anyString())).thenReturn(region);
+        List<Region> regions = asList(region);
+        when(regionRepository.findByDescriptionInIgnoreCase(any())).thenReturn(regions);
```

Comment:
I could not see any mock for these
`List<Region> findByRegionIdIn(List<String> regionId);

    List<Region> findByApiEnabled(boolean apiEnabled);`
    
   pact provider test will fail when the pact consumer tries to get these details as there is no mock data provided

Reply:
Addressed here: https://github.com/hmcts/rd-location-ref-api/pull/273/commits/69320723599354fac56ee17be201048157b4bee7